### ISSUE TITLE
fix overload warning from swig 4.0.1

### ISSUE THREAD

**Issue by burlen** [LBL-EESA/TECA#302]
```
/global/cscratch1/sd/loring/test_superbuild/TECA/python/../core/teca_variant_array.h:141: Warning 509: Overloaded method teca_variant_array::swap(p_teca_variant_array const &) effectively ignored,
/global/cscratch1/sd/loring/test_superbuild/TECA/python/../core/teca_variant_array.h:139: Warning 509: as it is shadowed by teca_variant_array::swap(teca_variant_array &).
/global/cscratch1/sd/loring/test_superbuild/TECA/python/../core/teca_variant_array.h:146: Warning 509: Overloaded method teca_variant_array::equal(const_p_teca_variant_array const &) const effectively ignored,
/global/cscratch1/sd/loring/test_superbuild/TECA/python/../core/teca_variant_array.h:144: Warning 509: as it is shadowed by teca_variant_array::equal(teca_variant_array const &) const.
```

**Comment by burlen** [comment]
fixed in #313

**Comment by burlen** [comment]
this was fixed. I think it went in with parallel collective I/O for cf writer
